src/wp-includes/global-styles-and-settings.php

```diff
@@ -255,3 +258,14 @@ function ( $item ) {
 		}
 	}
 }
+
+/**
+ * Private function to clean the caches under the theme_json group.
+ *
+ * @since 6.1.2
```

Comment:
Is this for 6.1.2 or 6.2.0?

Reply:
This PR should use 6.2.

https://github.com/WordPress/wordpress-develop/pull/3712#issuecomment-1385013665 for context. I guess the confusion comes from the fact that this PR may fix [an issue raised in 6.1](https://core.trac.wordpress.org/ticket/56970) for which we may (or may not) want to have a 6.1.2. But, yeah, that decision will happen in a different space, so 6.2 for now.